packages/jest-environment/src/index.ts

```diff
@@ -215,6 +215,11 @@ export interface Jest {
     moduleFactory: () => T | Promise<T>,
     options?: {virtual?: boolean},
   ): Jest;
+  /**
+   * Indicates that the module system should never return a mocked version of
+   * esm module (e.g. that it should always return the real module).
+   */
+  unstable_unmockModule(moduleName: string): Jest;
```

Comment:
Should be moved after `unmock` entry to keep the alphabetical order (ignoring `unstable_` part).